.github/workflows/perl-lint-checks.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: 'Perl static checks'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  perl-lint-checks:
+    runs-on: ubuntu-latest
+    name: "Perltidy"
+    container:
+      image: perldocker/perl-tester
+    steps:
+      - uses: actions/checkout@v4
+      - run: GITHUB_ACTIONS=1 ./tools/tidyall --check-only --all --quiet
```

Comment:
Where is `GITHUB_ACTIONS` being used? I don't see it anywhere...

Reply:
Glad you asked: deep in the source code of tidyall there's a not-so-fully documented usage of this:

https://github.com/houseabsolute/perl-code-tidyall/blob/1a75c32cd2d1a19b84eb1bdf2b8b2e2e733601a4/t/lib/TestFor/Code/TidyAll/DiffOnTidyError.pm#L24